From 9c470c0b158473e94cc725f4b40cfdf0284706b3 Mon Sep 17 00:00:00 2001
From: Swati Singh <quic_swasing@quicinc.com>
Date: Tue, 14 Feb 2023 17:56:39 +0530
Subject: [PATCH] net : Enable skb_recycler.

The changes makes skb_recycler_size as configurable.
If skb_recycler enabled, the size is set as,

	1. 1792 for Alder and Miami boards with
	   QSDK memory profile=512
	2. 2304 for all premium and
	   enterprize profiles.

Change-Id: Ic353cf494854b681556bb451ba860a0e4e615947
Signed-off-by: Swati Singh <quic_swasing@quicinc.com>
---
 net/Kconfig | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/net/Kconfig b/net/Kconfig
index 98939c20c162..d422dad56766 100644
--- a/net/Kconfig
+++ b/net/Kconfig
@@ -401,6 +401,13 @@ config SKB_RECYCLE_MAX_PREALLOC_SKBS
 	help
 	 Number of SKBs each of 4K size to be preallocated for recycling
 
+config SKB_RECYCLE_SIZE
+	int "SKB recycler size"
+	depends on SKB_RECYCLER
+	default 2304
+	help
+	SKB recycler default size
+
 config ALLOC_SKB_PAGE_FRAG_DISABLE
 	bool "Disable page fragment based skbuff payload allocations"
 	depends on !SKB_RECYCLER
-- 
2.34.1

